src/benchmark.rs

```diff
@@ -90,6 +90,28 @@ impl AsvCompare {
     }
 }
 
+pub async fn resolve_env(wd: &Path) -> Result<Vec<String>> {
+    tracing::info!("Resolving Environments: {:?}", wd);
+    let env = resolve_env_from_stdout(
+        Command::new("python")
+            .current_dir(wd)
+            .args([
+                "-c",
+                "import asv; import json; conf = asv.config.Config.load(\"asv.conf.json\"); env_names = [env.name for env in asv.environment.get_environments(conf, \"\")]; print(json.dumps(env_names))"
```

Comment:
Rust has multiline strings too!

You could assign it to a `const FOO: &'static str = ...` for nicer formatting.

An alternative would be to use [`include_str!`](https://doc.rust-lang.org/std/macro.include_str.html) to use a file as source and and still compile it into the binary.

```suggestion
                r#"
import asv
import json
conf = asv.config.Config.load("asv.conf.json")
env_names = [env.name for env in asv.environment.get_environments(conf, "")]
print(json.dumps(env_names))
"#
```

Reply:
Ah cool! will do `include_str!`